common/src/main/kotlin/tools/samt/common/Location.kt

```diff
@@ -1,15 +1,21 @@
 package tools.samt.common
 
 data class FileOffset(val charIndex: Int, val row: Int, val col: Int) {
-    override fun toString(): String = "$row:$col"
+    override fun toString(): String = "${row + 1}:${col + 1}"
 }
 
 data class Location(
-    val start: FileOffset,
-    val end: FileOffset = start,
+    var start: FileOffset,
+    var end: FileOffset = start,
```

Comment:
Why change to `var`?

Reply:
Temp fix below required it. Will be removed on the final version